include/trompeloeil.hpp

```diff
@@ -894,6 +908,13 @@ namespace trompeloeil
       reporter_obj()(s, file, line, msg);
     }
 
+  template <typename T>
+  void reporter<T>::
+    sendOk(char const *msg)
+    {
+        (void)msg;
+    }
```

Comment:
I think I prefer it if you never name the parameter, so you don't have to silence the warning.

```Cpp
sendOk(char const* /*msg*/)
{
}
```

Reply:
Will do